label_studio/ml_models/models.py

```diff
@@ -137,8 +146,16 @@ class ModelRunStatus(models.TextChoices):
     # todo may need to clean up in future
     @property
     def input_file_name(self):
-        return f'{self.project.id}_{self.model_version.pk}_{self.pk}/input_tasks.csv'
+        return f'{self.project.id}_{self.model_version.pk}_{self.pk}/input.csv'
 
     @property
     def output_file_name(self):
-        return f'{self.project.id}_{self.model_version.pk}_{self.pk}/output_tasks.csv'
+        return f'{self.project.id}_{self.model_version.pk}_{self.pk}/output.csv'
+
+    @property
+    def error_file_name(self):
+        return f'{self.project.id}_{self.model_version.pk}_{self.pk}/error.csv'
+
+    @property
+    def base_file_path(self):
+        return 's3://sandbox2-datasets-for-prompter-workflow/adala/hakan_test'   # TODO decide on path to use for LSE
```

Comment:
I figure we still need to determine this? Probably need to ask PLATE to set up the bucket?